apps/src/templates/levelSummary/SummaryTopLinks.jsx

```diff
@@ -0,0 +1,103 @@
+import React, {useCallback} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import i18n from '@cdo/locale';
+import styles from './check-for-understanding.module.scss';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+const SummaryTopLinks = ({
+  scriptData,
+  // redux
+  isRtl,
+  selectedSection,
+  currentLevelId,
+  levels,
+}) => {
+  const currentLevel = levels.find(l => l.activeId === currentLevelId);
+  const nextLevel = levels.find(l => l.position === currentLevel.position + 1);
+  const sectionParam = selectedSection?.id
+    ? `?section_id=${selectedSection.id}`
+    : '';
+
+  const logEvent = useCallback(eventName => {
+    const {level} = scriptData;
+    analyticsReporter.sendEvent(eventName, {
+      levelId: level.id,
+      levelName: level.name,
+      levelType: level.type,
+      ...scriptData.reportingData,
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
i'm not sure actually, this was there before. i'll look up the original commit and see if i can remember why i did this. it might also just be wrong

Reply:
It might be due to the eslint upgrades ([slack thread](https://codedotorg.slack.com/archives/C045UAX4WKH/p1680817629449789)). If possible, it would be great to clean this up while you're here but if it balloons for some reason, feel free to leave it

Reply:
ah yeah it [was](https://github.com/code-dot-org/code-dot-org/commit/7f4e69596f4d6c978619946bc1eb59931c4dc579#diff-362a2b7cbcac34cbcc681b6be0caf56fea454d78434bbab3c4fecd3135724d81). I added `scriptData` into the dependencies array and it seems happy now.

Reply:
thanks!